pom.xml

```
@@ -253,4 +259,16 @@
     <module>app</module>
     <module>benchmarks</module>
   </modules>
+
+  <profiles>
+    <profile>
+      <id>serverTest</id>
```

afaik this profile is not dedicated to the `test` target, so I'd name it like the module it is meant for.

```suggestion
      <id>server</id>
```